packages/server/graphql/public/types/AtlassianIntegration.ts

```diff
@@ -28,7 +34,18 @@ const AtlassianIntegration: AtlassianIntegrationResolvers = {
         projectKeyFiltersByCloudId[cloudId] = []
       })
     }
-    const issueRes = await manager.getIssues(queryString ?? null, isJQL, projectKeyFiltersByCloudId)
+    // Request one extra item to see if there are more results
+    const maxResults = first + 1
+    // Relay requires the cursor to be a string
+    const afterInt = parseInt(after ?? '-1', 10)
+    const startAt = Number.isNaN(afterInt) ? -1 : afterInt + 1
```

Comment:
+1 if `after === null` then `afterInt === -1` and `startAt === 0`.
If `after === 'invalid'` then `afterInt = NaN` and `startAt === -1`.
This seems a bit odd.

-1 I see we're using this value for cursor calculation, this will be wrong then.